src/main/java/org/jabref/gui/filelist/LinkedFilesEditDialogViewModel.java

```diff
@@ -102,19 +102,19 @@ public void setValues(LinkedFile entry) {
         }
     }
 
-    public StringProperty linkProperty() {
+    public StringProperty link() {
         return linkProperty;
```

Comment:
Sorry, I wasn't clear enough. The private field should be called `link` with public methods `linkProperty()` and `get/setLink`, with the latter optional.